src/AccessibilityInsights.Extensions.GitHub/PlaceHolderTextBox.cs

```diff
@@ -9,13 +9,13 @@ namespace AccessibilityInsights.Extensions.GitHub
 {
     public class PlaceHolderTextBox : TextBox
     {
-        private readonly string PlaceHolder = Properties.Resources.PlaceHolder;
-        private readonly SolidColorBrush BlackBrush = Application.Current.Resources["TextBrush"] as SolidColorBrush;
-        private readonly SolidColorBrush GrayBrush = Application.Current.Resources["TextBrushGray"] as SolidColorBrush;
+        public static string PlaceHolder { get; } = Properties.Resources.PlaceHolder;
+        public static SolidColorBrush BlackBrush { get; } = Application.Current.Resources["TextBrush"] as SolidColorBrush;
```

Comment:
>static [](start = 15, length = 6)

Making these static means that they only load once. What happens if the user changes color scheme after the static members are already initialized?